src/components/rhelGraphCard/rhelGraphCard.js

```diff
@@ -4,6 +4,7 @@ import numeral from 'numeral';
 import { Card, CardHead, CardActions, CardBody } from '@patternfly/react-core';
 import { Skeleton, SkeletonSize } from '@redhat-cloud-services/frontend-components';
 import { Select } from '../select/select';
+import { translate } from '../i18n/i18n';
```

Comment:
moving `translate` calls into here so that they could be called for other data arrays as well...and makes the chart helpers entirely generic...

Reply:
done!